src/translations/en/translations.json

```diff
@@ -35,6 +36,17 @@
   "New publication": "New publication",
   "New registration": "+ New registration",
   "New resource": "New resource",
+  "resource_form": {
```

Comment:
synes denne syntaxen er litt rar. Er det ikke bedre å skrive enten:
```
Resource Form: {
Title: Title
...
}
```

eller

```
resource_form: {
title: Title
...
}
```

?

Sånn at det blir konsistent?

Reply:
foreslår formen "resource_form.lovercase.greier" på alt ?

Reply:
greit for meg!